timrlink.net.CLI/Program.cs

```diff
@@ -131,33 +135,60 @@ private async Task ExportProjectTime(string connectionString, string from, strin
             var context = new DatabaseContext(new DbContextOptionsBuilder()
                 .UseSqlServer(connectionString)
                 .Options);
+            await InitializeDatabase(context);
 
-            var pendingMigrations = (await context.Database.GetPendingMigrationsAsync()).ToList();
-            if (pendingMigrations.Any())
-            {
-                var logger = LoggerFactory.CreateLogger<Program>();
-                logger.LogInformation($"Running Database Migration... ({string.Join(", ", pendingMigrations)})");
-                await context.Database.MigrateAsync();
-            }
-
-            await new ProjectTimeDatabaseExportAction(LoggerFactory, context, from: from, to: to, UserService,
-                TaskService, ProjectTimeService).Execute();
+            await new ProjectTimeDatabaseExportAction(LoggerFactory, context, from, to, UserService, TaskService,
+                ProjectTimeService).Execute();
         }
 
         private async Task ExportGroups(string connectionString)
         {
             var context = new DatabaseContext(new DbContextOptionsBuilder()
                 .UseSqlServer(connectionString)
                 .Options);
-            
+            await InitializeDatabase(context);
+
+            await new GroupUsersDatabaseExportAction(LoggerFactory, context, GroupService).Execute();
+        }
+
+        private async Task InitializeDatabase(DatabaseContext context)
+        {
             var pendingMigrations = (await context.Database.GetPendingMigrationsAsync()).ToList();
+            
             if (pendingMigrations.Any())
             {
-                Logger.LogInformation($"Running Database Migration... ({string.Join(", ", pendingMigrations)})");
-                await context.Database.MigrateAsync();
+                Logger.LogInformation("Running Database Migration... ({pendingMigrations})",
+                    string.Join(", ", pendingMigrations));
+                try
+                {
+                    await context.Database.MigrateAsync();
+                }
+                catch (SqlException exception)
```

Comment:
I would assume, that there also can be other `SqlException` which are not related to the assumed problem.

Can we concretize this a bit more?? And only do the following steps, when it's a clash of creating the concrete tables?